greysend_project/accounts/views.py

```diff
@@ -1,9 +1,8 @@
 from django.shortcuts import render
 from .models import AccountUser
-from django.contrib.auth.forms import UserCreationForm
 from django.contrib.auth.models import User
 from django.contrib.auth import login
-
+from forms import UserCreationForm
```

Comment:
this should be:
from django.contrib.auth.forms import UserCreationForm

Reply:
The line ```from django.contrib.auth.forms import UserCreationForm``` is in forms.py. I kind of envision forms.py as a central place for wrapping Django APIs.